src/lib.rs.in

```diff
@@ -650,16 +650,16 @@ class IronSdkAdvanced {
     private constructor = empty;
     /// Encrypt the provided document bytes. Return the encrypted document encryption keys (EDEKs) instead of creating a document entry in the IronCore webservice.
     /// 
-    /// @param a0 `documentData`   - bytes of the document to encrypt
-    /// @param a1 `encryptOpts`    - optional document encrypt parameters
-    method IronSdkAdvanced::document_encrypt_unmanaged(&self,_: &[i8], _: &DocumentEncryptOpts) -> Result<DocumentEncryptUnmanagedResult, String>; alias documentEncryptUnmanaged;
+    /// @param documentData   bytes of the document to encrypt
+    /// @param encryptOpts    optional document encrypt parameters
+    method IronSdkAdvanced::document_encrypt_unmanaged(&self, documentData: &[i8], encryptOpts: &DocumentEncryptOpts) -> Result<DocumentEncryptUnmanagedResult, String>; alias documentEncryptUnmanaged;
     /// Decrypt the provided encrypted document with the encrypted document encryption keys (EDEKs).
     /// 
-    /// @param a0 'encrypted_data' - bytes of encrypted document. Should be the same bytes returned from {@link documentEncryptUnmanaged(byte[], DocumentEncryptOpts)}
-    /// @param a1 'encrypted_deks' - encrypted document encryption keys. Should be the same edeks returned from {@link documentEncryptUnmanaged(byte[], DocumentEncryptOpts)}
+    /// @param encrypted_data bytes of encrypted document. Should be the same bytes returned from {@link #documentEncryptUnmanaged(byte[], DocumentEncryptOpts)}
```

Comment:
We should make the variable names consistently camel case since they're for java.

Reply:
Done